docs/development_guide.md

```diff
@@ -131,8 +204,11 @@ description of how you have tested your change. As a minimum you should have
 unit-test coverage for your change and make sure integration tests pass.
 
 ### Updating changes
-After making changes, you must again add, commit, and push those changes. Rather
-than creating new commits for related changes please run the following:
+
+After making changes, you must again add, commit, and push those changes.
+If you are making changes after a review process, you may create new commits for
+related changes by following the steps stated in "Pushing changes to your fork's
+branch". Otherwise, please run the following:
```

Comment:
Please add a note like: "Prefer to have one commit per each review round such that your reviewers can easily check what you have changed since last time."

Reply:
Done.